assets/styles/_custom-bootstrap.scss

```diff
@@ -15,6 +17,15 @@ $btn-padding-x: 0.5rem;
   }
 }
 
+// pagination overrides
+$pagination-color: #222;
+$pagination-active-bg: #f3453f;
+$pagination-active-color: $white;
+$pagination-hover-bg: #f3453f;
+$pagination-hover-color: $white;
+$pagination-border-color: $udb-grey;
+$pagination-focus-box-shadow: none;
```

Comment:
It looks similar, but in the old app the buttons were square (34x34 px) and a slightly smaller font-size. In the Vue app they look more rectangular (33x38 px) and have a bigger font-size (16px vs 15px).

AngularJS:
![Screenshot 2020-08-04 at 12 09 42](https://user-images.githubusercontent.com/959026/89282204-790ce180-d64b-11ea-8f9b-c0b11df8245a.png)

Vue:
![Screenshot 2020-08-04 at 12 10 04](https://user-images.githubusercontent.com/959026/89282216-7f02c280-d64b-11ea-9ff1-c77f65c78395.png)

Reply:
But if it's too hard to get to look more or less the same, leave it as it is.